Makefile

```diff
@@ -2,15 +2,17 @@
 
 changed_tests := `git status --porcelain | grep '^\(M\| M\|A\| A\)' | awk '{ print $$2 }' | grep '\/test_[a-zA-Z_\-\.]\+.py'`
 
-test: test-unit test-integration
+test:
+	@echo "Full test run starting..."
+	@time docker-compose run test tox
 
 test-unit:
 	@echo "Unit test run starting..."
-	tox -e unit-py27,unit-py35
+	@time docker-compose run test tox -e unit-py27,unit-py35,pep8
```

Comment:
That's not exactly right, since circle & appveyor don't use the Makefile to run tests -- they just call tox directly. The reason to do it this way is so that the tests always run in the same environment during development.